docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
+   @@ [2020-05-13T17:07:49][seconds=110] Info: Introspector and WebLogic pods with same namespace and domain-uid:
+   
+   NAME                              VERSION  IMAGE                    READY    PHASE      
+   --------------------------------  -------  -----------------------  -------  ---------  
+   'sample-domain2-admin-server'     '1'      'model-in-image:WLS-v1'  'true'   'Running'  
+   'sample-domain2-managed-server1'  '1'      'model-in-image:WLS-v1'  'false'  'Running'  
+   'sample-domain2-managed-server2'  '1'      'model-in-image:WLS-v1'  'false'  'Pending'  
+   
+   @@ [2020-05-13T17:07:50][seconds=111] Info: '1' WebLogic pods currently match all criteria, expecting '3'.
+   @@ [2020-05-13T17:07:50][seconds=111] Info: Introspector and WebLogic pods with same namespace and domain-uid:
+   
+   NAME                              VERSION  IMAGE                    READY    PHASE      
+   --------------------------------  -------  -----------------------  -------  ---------  
+   'sample-domain2-admin-server'     '1'      'model-in-image:WLS-v1'  'true'   'Running'  
+   'sample-domain2-managed-server1'  '1'      'model-in-image:WLS-v1'  'false'  'Running'  
+   'sample-domain2-managed-server2'  '1'      'model-in-image:WLS-v1'  'false'  'Running'  
+   
+   @@ [2020-05-13T17:08:32][seconds=153] Info: '2' WebLogic pods currently match all criteria, expecting '3'.
+   @@ [2020-05-13T17:08:32][seconds=153] Info: Introspector and WebLogic pods with same namespace and domain-uid:
+   
+   NAME                              VERSION  IMAGE                    READY    PHASE      
+   --------------------------------  -------  -----------------------  -------  ---------  
+   'sample-domain2-admin-server'     '1'      'model-in-image:WLS-v1'  'true'   'Running'  
+   'sample-domain2-managed-server1'  '1'      'model-in-image:WLS-v1'  'true'   'Running'  
+   'sample-domain2-managed-server2'  '1'      'model-in-image:WLS-v1'  'false'  'Running'  
+   
+   @@ [2020-05-13T17:08:34][seconds=155] Info: '3' WebLogic pods currently match all criteria, expecting '3'.
+   @@ [2020-05-13T17:08:34][seconds=155] Info: Introspector and WebLogic pods with same namespace and domain-uid:
+   
+   NAME                              VERSION  IMAGE                    READY   PHASE      
+   --------------------------------  -------  -----------------------  ------  ---------  
+   'sample-domain2-admin-server'     '1'      'model-in-image:WLS-v1'  'true'  'Running'  
+   'sample-domain2-managed-server1'  '1'      'model-in-image:WLS-v1'  'true'  'Running'  
+   'sample-domain2-managed-server2'  '1'      'model-in-image:WLS-v1'  'true'  'Running'  
+   
+   
+   @@ [2020-05-13T17:08:34][seconds=155] Info: Success!
+
+   ```
+   {{% /expand%}}
+
+1. After the `sample-domain2` domain is running, you can call its sample web application to verify that it's fully active.
+
+   Send a web application request to the Ingress controller for `sample-domain2`:
```

Comment:
Ingress controller -> ingress controller (Capitalization change to match Kubernetes docs)

Reply:
done